packages/url/init.lua

```diff
@@ -46,8 +46,11 @@ function package.declareSettings (_)
 end
 
 function package:registerCommands ()
-
   self:registerCommand("href", function (options, content)
+    if (not content or #content == 0 and not content[1]) and options.src then
+      content = {options.src}
```

Comment:
So this would lead to have both a content and a src, which in this case will bypass the inputfilter for extra breakpoints. Shouldn't be `options.src` set to nil if this is moved to the content, so that your new `\href[src=URL]` becomes really identical to `\href{URL}`?

Reply:
Additionaly, what happens if one just calls `\href`? (no content, no src option)